.github/workflows/test_rpm_from_scratch_process.yml

```diff
@@ -0,0 +1,48 @@
+name: RPM Build Script Test
+
+on:
+  pull_request:
+
+permissions: {}
+jobs:
+  Run_in_docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: ["rockylinux:8.6",
+             "rockylinux:8.8",
+             "rockylinux:8.10",
+             "rockylinux:9.2",
+             "rockylinux:9.3",
+             "opensuse/leap:15.4",
+             "opensuse/leap:15.5",
+             "opensuse/leap:15.6"]
```

Comment:
I wonder if using our self-hosted runner and therefore our own docker hub/cache, you will find more of these older versions.  @JohnMalmberg Would we have these old point releases in our docker runners?

Reply:
On the docker runners we are only keeping rockylinux8 and 9 up to date, no dot releases.
We have opensuse/leap:15.[2-5] and opensuse/leap-dnf:15.[4-5]

Just updated so that soon they will have leap and leap-dnf 15.6 and Fedora:41

Reply:
> On the docker runners we are only keeping rockylinux8 and 9 up to date, no dot releases.

This is a good case why it would be useful if we were keeping the EL point releases in our docker hub.